camera_calibrator_extrinsic: # Find your parameters by running the intrinsic calibration first!
  ros__parameters:
    calibration_matrix:
      - 604.71960668
      - 0.0
      - 680.80027539
      - 0.0
      - 522.46694113
      - 624.75623441
      - 0.0
      - 0.0
      - 1.0
    dist_coefficients:
      - -0.10623745
      - 0.0052871
      - -0.05780306
      - 0.00980659
      - 0.04443656
      - 0.11454198
      - 0.03534485
      - -0.10617324
      - -0.0195471
      - 0.00030371
      - -0.00527497
      - 0.03740825